lib/OpenQA/Schema.pm

```diff
@@ -100,19 +87,16 @@ sub deploy {
 }
 
 # Class attribute used for testing with OpenQA::Test::Database
-sub search_path_for_tests {
-    my $class = shift;
+sub search_path_for_tests ($class, $new_search_path = undef) {
     state $search_path;
-    $search_path = shift if @_;
+    $search_path = $new_search_path if defined $new_search_path;
```

Comment:
There's no way to set `$search_path` to undef now.

Reply:
thx, I have reverted in the meantime and will look into doing this properly again

Reply:
What do you suggest we do here instead?

Reply:
```perl
sub search_path_for_tests ($class, @args) {
    state $search_path;
    $search_path = shift @args if @args;
```

Reply:
applied in
* #4927